Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM alpine:3.7
 
 RUN apk add --no-cache ca-certificates cdrkit
 
-COPY machine-controller /usr/local/bin
-COPY webhook /usr/local/bin
+COPY machine-controller machine-controller-userdata-* webhook /usr/local/bin
```

Comment:
The `/usr/local/bin` need a trailing `/`, otherwise this errors. This is also the reason the e2e tests fail

Reply:
Grmpflx! 🤬 Yes, already once stumbled about it and didn't remembered it here. Gnah! Good finding.